### ISSUE TITLE
Allow to provide non-default path to project .git folder

### ISSUE THREAD

**Issue by Tapchicoma** [JLLeitschuh/ktlint-gradle#284]
Followup of #101:
Current implementation only adds a git hook, when `.git/` folder is located in Gradle root project folder. But quite often project has non-gradle top folder. Plugin should add a way to define non-standard `./git` folder location.

**Comment by tasomaniac** [comment]
I just tried this and hit the same problem. 2 points to add: 

- Even if I moved the pre_commit hook into the correct `.git/` folder, the relative paths of the changed files also need to be adjusted according to the root folder of the project. Right not it cannot find the file locations.

- Secondly, `grgit` is a really good library to deal with Git operations in Gradle. It can find the closest git to the current path (that's also how regular command line git works). So without the custom property in the plugin, we can detect the correct `.git` folder.
